main.tf

```diff
@@ -58,23 +68,24 @@ data "aws_acm_certificate" "ssl" {
   statuses = ["ISSUED"]
 }
 
+resource "aws_cloudfront_origin_access_identity" "orig_access_ident" {
+  comment = "CloudFront Origin Access Identity to access S3 Bucket ${local.www_domain}"
+}
+
 resource "aws_cloudfront_distribution" "cdn" {
   count               = "${length(local.domains)}"
   enabled             = true
   default_root_object = "${element(local.domains, count.index) == local.www_domain ? "index.html" : ""}"
   aliases             = ["${element(local.domains, count.index)}"]
   is_ipv6_enabled     = true
+  http_version        = "http2"
```

Comment:
This is the default value so it can be omitted.

Reply:
Sure I will remove this